docs/data/joy/customization/theme-typography/theme-typography.md

```diff
@@ -0,0 +1,162 @@
+  ```
+
+- [CSS Baseline](/joy-ui/react-css-baseline/) component: by default, it applies the `body1` level to the global stylesheet:
+
+  ```jsx
+  <CssBaseline />
+
+  // inherits the `theme.typography.body1` styles
+  <p>Hello World</p>
+  ```
+
+- [`sx`](/joy-ui/customization/approaches/#sx-prop) prop: use `typography: $level` to get the specific theme typography level:
+
+  ```jsx
+  // to apply the `theme.typography.body2` styles:
+  <Box sx={{ typography: 'body2' }}>Small text</Box>
+  ```
+
+- [`styled`](/joy-ui/customization/approaches/#reusable-component): create a custom component and apply the style from `theme.typography.*` directly:
+
+  ```jsx
+  import { styled } from '@mui/joy/styles';
+
+  const Tag = styled('span')((theme) => ({
+    ...theme.typography.body2,
+    color: 'inherit',
+    borderRadius: theme.vars.radius.xs,
+    boxShadow: theme.vars.shadow.sm,
+    padding: '0.125em 0.375em',
+  }));
+  ```
+
+## Common examples
+
+Here is a collection of well-known typography systems that you can use with Joy UI.
+
+### Fluent (Microsoft)
+
+- Design resource: [Figma](https://www.figma.com/community/file/836828295772957889)
+- Font: [Segoe UI](https://learn.microsoft.com/en-us/typography/font-list/segoe-ui)
+
+<iframe src="https://codesandbox.io/embed/joy-ui-fluent-typography-system-j86fct?module=%2Fdemo.tsx&fontsize=14&hidenavigation=1&theme=dark&view=preview"
+     style="width:100%; height:360px; border:0; border-radius: 4px; overflow:hidden;"
+     title="Joy UI - Fluent Typography System"
+     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+   ></iframe>
+
+### Human Interface Guidelines (Apple)
+
+- Design resource: [Sketch library](https://developer.apple.com/design/resources/)
+- Font: [San Francisco (SF)](https://developer.apple.com/fonts/)
+
+<iframe src="https://codesandbox.io/embed/joy-ui-human-interface-guidelines-typography-system-lkuz4d?module=%2Fdemo.tsx&fontsize=14&hidenavigation=1&theme=dark&view=preview"
+     style="width:100%; height:320px; border:0; border-radius: 4px; overflow:hidden;"
+     title="Joy UI - Human Interface Guidelines Typography System"
+     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+   ></iframe>
+
+### Material Design 3 (Google)
+
+- Design resource: [Figma](https://www.figma.com/community/file/1035203688168086460)
+- Font: [Roboto](https://fonts.google.com/specimen/Roboto)
+
+<iframe src="https://codesandbox.io/embed/joy-ui-material-3-typography-system-lx044f?module=%2Fdemo.tsx&fontsize=14&hidenavigation=1&theme=dark&view=preview"
```

Comment:
Let's please use `{{"demo": "iframe": true }}` the next time for this type of demos. It's a pain to keep in sync in https://codesandbox.io/dashboard/sandboxes/used%20in%20docs?workspace=836800c1-9711-491c-a89b-3ac24cbd8cd8 and it's slower to load than it needs to be for developers.